types/index.d.ts

```diff
@@ -909,6 +909,15 @@ declare namespace RamdaAdjunct {
         then<T>(onFulfilled: Function, thenable: Promise<T>): Promise<T>;
         then<T>(onFulfilled: Function): (thenable: Promise<T>) => Promise<T>;
 
+        /**
+         * Composable shortcut for `Promise.then` that allows for success and failure call backs.
+         * The thenCatchP function returns a Promise. It takes three arguments: a callback function for the success of the Promise,
+         * a callback function for the failure of the Promise, and the promise instance itself.
+         */
```

Comment:
We also need a tests for these typings. Please have a look at how to do it: https://github.com/char0n/ramda-adjunct/tree/054e9aa47bf334cc664a4e56ec896fbc184b28b1/types/test

Reply:
Will do, thanks!